frame/contracts/CHANGELOG.md

```diff
@@ -0,0 +1,78 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+The semantic versioning garantuees do cover the interface to the substrate runtime which
```

Comment:
```suggestion
The semantic versioning guarantees do cover the interface to the substrate runtime which
```

Reply:
Also do you mean *do not*?

Reply:
No I think it is correct as I have written it down: It  is covered as the runtime declares pallet_contracts as dependency in its Cargo.toml. Not sure why you think I mean the opposite. Should that be expressed in a different way then?

Reply:
No it was just because of the word `do` on its own, rather than the content of the sentence, so was just checking.

Reply:
I removed the word.